app/javascript/flavours/glitch/components/hashtag_bar.tsx

```diff
@@ -0,0 +1,234 @@
+import { useState, useCallback } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import { Link } from 'react-router-dom';
+
+import type { List, Record } from 'immutable';
+
+import { groupBy, minBy } from 'lodash';
+
+import { getStatusContent } from './status_content';
+
+// About two lines on desktop
+const VISIBLE_HASHTAGS = 7;
```

Comment:
Could this be an app setting or otherwise configurable?

Reply:
This does seem like the type of thing that would be useful to have as a config option, yeah.